compair/api/answer.py

```diff
@@ -117,15 +123,29 @@ def get(self, course_uuid, assignment_uuid):
                 Answer.active == True,
                 Answer.practice == False,
                 Answer.draft == False,
-                UserCourse.course_role != CourseRole.dropped
+                or_(
+                    and_(UserCourse.course_role != CourseRole.dropped, Answer.user_id != None),
+                    Answer.group_id != None
+                )
```

Comment:
What if it is a group assignment but everyone in the group dropped the course?

Reply:
I think its still fair to show the group's answer in this case though it could be a bit odd (very edge case). The current way it is setup, the instructor can also create an empty group and submit an answer on behalf of it (even though there are no students/users in it). 

It might be best to revisit later if it flagged by anyone as problematic?

Reply:
Agree.  In such rare case, if instructors dont want to show the answers, they could just delete them.